packages/flutter/lib/src/widgets/fade_in_image.dart

```diff
@@ -61,28 +62,21 @@ import 'ticker_provider.dart';
 /// )
 /// ```
 /// {@end-tool}
-class FadeInImage extends StatefulWidget {
-  /// Creates a widget that displays a [placeholder] while an [image] is loading
-  /// then cross-fades to display the [image].
+class FadeInImage extends StatelessWidget {
+  /// Creates a widget that displays a [placeholder] while an [image] is loading,
+  /// then fades-out the placeholder and fades-in the image.
   ///
   /// The [placeholder], [image], [fadeOutDuration], [fadeOutCurve],
   /// [fadeInDuration], [fadeInCurve], [alignment], [repeat], and
   /// [matchTextDirection] arguments must not be null.
   ///
-  /// There are two different semantic label for the class.
-  /// [placeholderSemanticLabel] is used for defining a semantics label for
-  /// [placeholder]. [imageSemanticLabel] is used for defining a semantics label
-  /// for [image]
-  ///
-  /// If [excludeFromSemantics] is true, then [placeholderSemanticLabel] and
-  /// [imageSemanticLabel] will be ignored.
+  /// If [excludeFromSemantics] is true, then [imageSemanticLabel] will be ignored.
```

Comment:
@jonahwilliams Do you remember what the use case for adding the placeholderSemanticsLabel was?

Reply:
The idea was that the unloaded in image might require a separate label, if that label is external.

Example: "loading" - > "Foo name"

Reply:
Looks like it was added in https://github.com/flutter/flutter/pull/28799 just for the sake of completeness (the bug it fixed was the lack of any semantic label support in FadeInImage).

I think it's ill-advised to have a separate semantic label while the image is loading.  I'm gonna comment on the original PR to see if they object to its removal.

Reply:
Your argumentation SGTM. Unless somebody objects, let's remove it. (Can you update the PR description to reflect this breaking change for future archeologists?).

Reply:
Done